### PR TITLE
Allow other die types

### PR DESCRIPTION
**Pull request by iamevn**
Allow arbitrary positive integer die types and add default display for when there's no matching emoji.

This at least partially addresses #334

### CODE REVIEW

File: packages/aquarius/src/bot/commands/roll.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ export default async ({ aquarius, analytics }) => {
 
     const values = Array.from(
       roll.matchAll(
-        /(?:(?<sign>[+-]) )?(?<dieCount>\d+)?d(?<dieType>100|20|12|10|8|6|4)(?: ?(?<type>[+-]) ?(?<modifier>\d+(?!d)))?/gi
+        /(?:(?<sign>[+-]) )?(?<dieCount>\d+)?d(?<dieType>[1-9]\d*)(?: ?(?<type>[+-]) ?(?<modifier>\d+(?!d)))?/gi
```
Comment (iamevn) [inline packages/aquarius/src/bot/commands/roll.js:49]: If you want to allow leading zeros, `(?<dieType>[1-9]\d*)` would become `(?<dieType>0*[1-9]\d*)`

File: packages/aquarius/src/bot/commands/roll.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ export default async ({ aquarius, analytics }) => {
         const emoji =
           dieType === '100'
             ? `${aquarius.emojiList.get('d10')}${aquarius.emojiList.get('d10')}`
-            : aquarius.emojiList.get(`d${dieType}`);
+            : (aquarius.emojiList.get(`d${dieType}`) || `d${dieType}`);
```
Comment (iamevn) [inline packages/aquarius/src/bot/commands/roll.js:62]: Added parentheses not necessary but I can never remember exact precedence rules and they're only one level apart anyways.

File: packages/aquarius/src/bot/commands/roll.js (hunk 1 above)
Comment (iamevn) [reply packages/aquarius/src/bot/commands/roll.js:49]: You'd also want to strip the zeros before emojifying them so probably not worth the effort

File: packages/aquarius/src/bot/commands/roll.js (hunk 2 above)
Comment (IanMitchell) [reply packages/aquarius/src/bot/commands/roll.js:62]: Looks like the code formatted would have added them too!

**Pull request merged**